src/sentry/profiles/tasks.py

```diff
@@ -0,0 +1,124 @@
+from typing import Any, MutableMapping
+
+from django.conf import settings
+
+from sentry.lang.native.symbolicator import Symbolicator
+from sentry.models import Project
+from sentry.profiles.device import classify_device
+from sentry.tasks.base import instrumented_task
+from sentry.utils import json, kafka_config
+from sentry.utils.pubsub import KafkaPublisher
+
+processed_profiles_publisher = None
+
+
+@instrumented_task(  # type: ignore
+    name="profiles.process",
+    queue="profiles.process",
+    default_retry_delay=5,
+    max_retries=5,
```

Comment:
You'll want `acks_late=True` on this task. Without that celery acks the message when the job starts, instead of when it ends. Acking at the start means any deploy related restarts will drop work. With acks_late the job will be retried should it be killed by a deploy.